backend/app/services/llm/jobs.py

```
@@ -0,0 +1,162 @@
+        )
+
+    return callback_response.model_dump()
+
+
+def execute_job(
+    request_data: dict,
+    project_id: int,
+    organization_id: int,
+    job_id: str,
+    task_id: str,
+    task_instance,
```

Ruff (0.14.2)

Unused function argument: `task_instance`

(ARG001)